entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -56,7 +56,20 @@ Three files are added to your project under the **Migrations** directory:
 The timestamp in the filename helps keep them ordered chronologically so you can see the progression of changes.
 
 > [!TIP]
-> You are free to move Migrations files and change their namespace. New migrations are created as siblings of the last migration.
+> You are free to move Migrations files and change their namespace manually. New migrations are created as siblings of the last migration.
+> 
+> Alternatively you can use `-Namespace` (PMC) or `--namespace` (CLI) to specify the namespace at generation time.
```

Comment:
Spell out PMC and CLI to help localization

Reply:
This is how its referred to elsewhere throughout the docs.

Reply:
Then change it everywhere

Reply:
Hmmm. I'll do it here, but doing it everywhere is a big change - and I'm not even sure it would be appropriate everywhere - it's in tab titles and in places where the abbreviations have been specifically called out because they're used so much. I'll raise another bug to discuss with the team.

Reply:
Raised #2335